src/main/resources/Confluence/Macros/Tip.xml

```diff
@@ -0,0 +1,370 @@
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </contentType>
+      <defaultCategory>
+        <disabled>0</disabled>
+        <name>defaultCategory</name>
+        <number>4</number>
+        <prettyName>Default category</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </defaultCategory>
+      <description>
+        <contenttype>PureText</contenttype>
+        <disabled>0</disabled>
+        <editor>PureText</editor>
+        <name>description</name>
+        <number>3</number>
+        <prettyName>Macro description</prettyName>
+        <rows>5</rows>
+        <size>40</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.TextAreaClass</classType>
+      </description>
+      <id>
+        <disabled>0</disabled>
+        <name>id</name>
+        <number>1</number>
+        <prettyName>Macro id</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </id>
+      <name>
+        <disabled>0</disabled>
+        <name>name</name>
+        <number>2</number>
+        <prettyName>Macro name</prettyName>
+        <size>30</size>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.StringClass</classType>
+      </name>
+      <supportsInlineMode>
+        <disabled>0</disabled>
+        <displayFormType>select</displayFormType>
+        <displayType>yesno</displayType>
+        <name>supportsInlineMode</name>
+        <number>5</number>
+        <prettyName>Supports inline mode</prettyName>
+        <unmodifiable>0</unmodifiable>
+        <classType>com.xpn.xwiki.objects.classes.BooleanClass</classType>
+      </supportsInlineMode>
+      <visibility>
+        <cache>0</cache>
+        <disabled>0</disabled>
+        <displayType>select</displayType>
+        <freeText>forbidden</freeText>
+        <largeStorage>0</largeStorage>
+        <multiSelect>0</multiSelect>
+        <name>visibility</name>
+        <number>6</number>
+        <prettyName>Macro visibility</prettyName>
+        <relationalStorage>0</relationalStorage>
+        <separator>|</separator>
+        <separators>|</separators>
+        <size>1</size>
+        <unmodifiable>0</unmodifiable>
+        <values>Current User|Current Wiki|Global</values>
+        <classType>com.xpn.xwiki.objects.classes.StaticListClass</classType>
+      </visibility>
+    </class>
+    <property>
+      <async_cached>0</async_cached>
+    </property>
+    <property>
+      <async_context/>
+    </property>
+    <property>
+      <async_enabled>0</async_enabled>
+    </property>
+    <property>
+      <code>{{velocity}}
+  {{success}}
+    #if("$!wikimacro.parameters.title" != "")
+    **$!{wikimacro.parameters.title}**
+
+    #end
+    $wikimacro.content
```

Comment:
This means we won't be able to use Velocity macro inside the Tip macro content, unless you wrap the Tip macro with the Velocity macro. Maybe that's fine, just wan to make sure you are aware.

Reply:
Couldn't you use ``{{wikimacrocontent/}}``?

Reply:
About the velocity macro: I hope that content migrated from Confluence does not have velocity macros embedded in their "infoboxes" macros. Then maybe instead of writing these wrapper macros we might better add them to the macro-mapping in the Confluence Migrator (if that macro mapping has been implemented, I forgot to check the current state, sorry.) That might avoid using velocity at all.

Reply:
I have updated the PR :  the "tip" marco (also also the "note" macro) now uses the ``{{wikimacrocontent/}}``, and also ``{{wikimacroparameter name="title" /}}``.

The latter inserts an empty line on top of the title, however.
Old version without ``wikimacroparameter``:
![tip-nowikiparam](https://user-images.githubusercontent.com/4411046/180000762-46266e4a-2767-457a-b110-297f2f585bab.png)

New version with ``wikimacroparameter``:
![tip-wikiparam](https://user-images.githubusercontent.com/4411046/180000921-379157c8-5c82-424c-9347-20ac2625d7b8.png)

I think I can hush this up via CSS if necessary, but that might be ugly :-/